src/Avalonia.Controls/Notifications/INotificationManager.cs

```diff
@@ -13,6 +14,18 @@ public interface INotificationManager
         /// Show a notification.
         /// </summary>
         /// <param name="notification">The notification to be displayed.</param>
-        void Show(INotification notification);
+        /// <param name="cancellationToken">The cancellation token.</param>
+        void Show(INotification notification, CancellationToken cancellationToken = default);
```

Comment:
This one is going to be a bigger breaking change, than just adding methods.
Since old IL references to Show method will be binary broken.

Reply:
In other places we have added new interfaces, like INotificationManager2, with new members. With a "TODO12 Merge with INotificationManager" comment.

Reply:
INotificationManager2 interface should remove any api breaking changes.

Reply:
I can remove cancellationToken and do it for 12.0 its not deal breaker

Reply:
https://github.com/AvaloniaUI/Avalonia/pull/15628/commits/e09d0aadeb709b2ed24932e19391652eae1cffb3